src/sections/Meshmap/meshmap_features.js

```diff
@@ -0,0 +1,267 @@
+        font-size: 17px;
+        font-weight:400;
+        border-color: #ffffff
+    }
+    `;
+
+function MeshmapFeatures() {
+
+    const handleVisualize = () => {
+        setFeature('Visualize');
+        setFeatureDesc('Apply patterns and manage many Kubernetes clusters.');
+        setDesc1("Deploy designs, apply patterns, manage and operate your deployments in real-time. Bring all your Kubernetes clusters under a common point of management. Interactively connect to terminal sessions or initiate and search log streams from your containers.");
+        setfeatureImage(VisualizerIcon);
+    }
+    const handleDesign = () => {
+        setFeature('Design');
+        setFeatureDesc('Discover and model cloud native deployments.'); 
+        setDesc1("Discover and model cloud native deployments.Drag-and-drop your cloud native infrastructure using a pallete of thousands of versioned Kubernetes components. Using GitOps? Integrate advanced performance analysis into your pipeline.");
+        setfeatureImage(DesignerIcon);
+    }
+    const handleCollaborate = () => {
+        setFeature('Collaborate');
+        setDesc1("Friends don't let friends GitOps alone!");
+        setfeatureImage(collaboration);
+    }
+
+    const [feature, setFeature] = React.useState('Design');
+    const [featureDesc, setFeatureDesc] = React.useState('Discover and model cloud native deployments.')
+    const [desc1, setDesc1] = React.useState("Discover and model cloud native deployments.Drag-and-drop your cloud native infrastructure using a pallete of thousands of versioned Kubernetes components. Using GitOps? Integrate advanced performance analysis into your pipeline.");
+    const [featureImage, setfeatureImage] = React.useState(DesignerIcon)
+
+    return (
+        <MeshmapFeaturesWrapper>
+            <div className="circles-icon">
+                <img src={circles} alt="" />
+            </div>
+            <div className='designer-logo'>
+                <img src={featureImage} alt="Designer" />
+                <img src={contextAware} style={{marginTop : "6rem"}} alt="Designer" />
+
+            </div>
+            <div className="grid-icon">
+                <img src={grid} alt="grid" />
+            </div>
+            <div className="tagline">
+                MeshMap Features
+            </div>
+            <div className="point"></div>
+            <div className="features-point" onClick={handleDesign}>Design</div>
+            <div className="vertical-line"></div>
+            <div className="point" style={{ top: "76.8rem" }}></div>
+            <div className="features-point" style={{ top: "76rem", left: "9rem" }} onClick={handleVisualize}>Visualize</div>
+            <div className="vertical-line" style={{ top: "78rem" }}></div>
+            <div className="point" style={{ top: "94.8rem" }}></div>
+            <div className="features-point" style={{ top: "94rem", left: "8rem" }} onClick={handleCollaborate}>Collaborate</div>
```

Comment:
Please use flex, grid system. This type of inline styling, top and left will create issues.

Reply:
Okay. I'll take care of that